src/components/victory-errorbar/errorbar.js

```diff
@@ -0,0 +1,127 @@
+import React, { PropTypes } from "react";
+import { isArray } from "lodash";
+
+export default class ErrorBar extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.renderBorder = this.renderBorder.bind(this);
+  }
+
+  static propTypes = {
+    index: React.PropTypes.number,
+    datum: PropTypes.object,
+    events: PropTypes.object,
+    scale: PropTypes.object,
+    style: PropTypes.object,
+    x: PropTypes.number,
+    y: PropTypes.number,
+    errorX: PropTypes.oneOfType([
+      PropTypes.number,
+      PropTypes.array
+    ]),
+    errorY: PropTypes.oneOfType([
+      PropTypes.number,
+      PropTypes.array
+    ]),
+    borderWidth: PropTypes.number
+  };
+
+  renderBorder() {
+    const {
+      x,
+      y,
+      errorX,
+      errorY,
+      borderWidth
+    } = this.props;
+
+    return (
+      <g>
+        <line
+          {...this.props.events}
+          style={this.props.style}
+          x1={x + (isArray(errorX) ? errorX[1] : errorX)}
+          x2={x + (isArray(errorX) ? errorX[1] : errorX)}
+          y1={y - borderWidth}
+          y2={y + borderWidth}
+        />
+        <line
+          {...this.props.events}
+          style={this.props.style}
+          x1={x - (isArray(errorX) ? errorX[1] : errorX)}
+          x2={x - (isArray(errorX) ? errorX[1] : errorX)}
+          y1={y - borderWidth}
+          y2={y + borderWidth}
+        />
+        <line
+          {...this.props.events}
+          style={this.props.style}
+          x1={x - borderWidth}
+          x2={x + borderWidth}
+          y1={y - (isArray(errorY) ? errorY[1] : errorY)}
+          y2={y - (isArray(errorY) ? errorY[1] : errorY)}
+        />
+        <line
+          {...this.props.events}
+          style={this.props.style}
+          x1={x - borderWidth}
+          x2={x + borderWidth}
+          y1={y + (isArray(errorY) ? errorY[1] : errorY)}
+          y2={y + (isArray(errorY) ? errorY[1] : errorY)}
+        />
+      </g>
+    );
+  }
+
+  render() {
+    const {
+      x,
+      y,
+      errorX,
+      errorY
+    } = this.props;
+
+    return (
+      <g>
+        {this.renderBorder()}
+        <line
```

Comment:
Can this be extracted into another named method? Also with how similar all these lines are it _seems_ like this could be refactored into a couple general helpers, but it might be more trouble than it's worth.